install.sh

```
@@ -34,43 +34,90 @@ fi
+
+    # Set GENESIS_TOOLS_PATH as a persistent user environment variable
+    if setx GENESIS_TOOLS_PATH "$WIN_DIR" > /dev/null 2>&1; then
+        echo "📝 Set GENESIS_TOOLS_PATH=$WIN_DIR"
+    else
+        echo "❌ Failed to set GENESIS_TOOLS_PATH via setx"
+        exit 1
+    fi
+
+    # Add to user PATH if not already present (case-insensitive check)
+    CURRENT_PATH="$(powershell.exe -NoProfile -Command "[Environment]::GetEnvironmentVariable('Path', 'User')" 2>/dev/null | tr -d '\r')"
+    if echo "$CURRENT_PATH" | grep -qi "$(echo "$WIN_DIR" | sed 's/\\/\\\\/g')"; then
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)